src/worker.cc

```diff
@@ -152,7 +152,8 @@ const std::unordered_map<int, std::string> warning_codes = {
   {201, R"("sources" have date_time set, but "arrive_by" was requested, ignoring date_time)"},
   {202, R"("targets" have date_time set, but "depart_at" was requested, ignoring date_time)"},
   {203, R"("waiting_time" is set on a location of type "via" or "through", ignoring waiting_time)"},
-  {204, R"("exclude_polygons" received invalid input, ignoring exclude_polygons)"}
+  {204, R"("exclude_polygons" received invalid input, ignoring exclude_polygons)"},
+  {205, R"("disable_hierarchy_pruning" received invalid input, max distance exceeded, ignoring disable_hierarchy_pruning)"}
```

Comment:
good that you added a warning (it wasn't part of my description I think), but makes very much sense.

nit:
```suggestion
  {205, R"("disable_hierarchy_pruning" exceeded the max distance , ignoring disable_hierarchy_pruning)"}
```

Reply:
Thanks for the revision. done!